src/VisualStudio/CSharp/Test/PersistentStorage/AbstractPersistentStorageTests.cs

```diff
@@ -30,9 +29,12 @@ public enum Size
         {
             Small,
             Medium,
-            Large
+            Large,
+            ExtraLarge,
         }
 
+        private const int Iterations = 100;
```

Comment:
note: even running all tests 100 times each, we still run all sqlite and cloudcache tests in under 30s total.  so this seems worth it for the coverage.